cluster/ci/origin/pipelines/test-build/Jenkinsfile

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env groovy
```

Comment:
feels like this whole pipe should just be one parallel step in the main build

Reply:
we'll just follow the pattern of launching the build from whatever test requires it.

Reply:
Don't we launch the main build from whatever test requires it already?